core/templates/pages/preferences-page/preferences-page.component.ts

```diff
@@ -175,18 +192,53 @@ export class PreferencesPageComponent {
     }
   }
 
+  private saveProfileImage(image: Blob): void {
```

Comment:
Bit confused about the two options here. Could you elaborate on when each is needed/used?

Reply:
`saveProfileImageToLocalStorage` is used when Oppia developers are running the dev server. As a note: when adding images in exploration content, we have a similar set of two options (`saveImageToLocalStorage` and `postImageToServer`).